usagov/vote-gov-drupal
# Feature/vote 879 state page styling

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-897

## Description
Styling of all state page and it's components

### Post-deploy steps
1. Run lando retune
2. Run npm run build in the theme folder

### Testing steps
1. Create a state page in the admin interface with at least two NVRF links for the selector
2. Verify the selector works as intended in the browser
3. Verify that the pages are styled correctly
4. Change the registration types in the admin interface and observe that they are shown/hidden on the page as the data changes
